application/helpers/update/updates/Update_628.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use LimeSurvey\Helpers\Update\DatabaseUpdateBase;
+
+class Update_628 extends DatabaseUpdateBase
+{
+    /**
+     * @inheritDoc
+     */
+    public function up()
+    {
```

Comment:
I was expecting here to migrate current settings in config to user settings, shouldn't we to avoid loss of data? (I know, not so critical, so not sure it is worth it)

Reply:
> I was expecting here to migrate current settings in config to user settings, shouldn't we to avoid loss of data? (I know, not so critical, so not sure it is worth it)

I start with a solution to migrate (only surey since question are not saved) but https://bugs.limesurvey.org/view.php?id=19904#c82083

And the solution must be tested on MS et PG.